ephemeris/shed_tools.py

```diff
@@ -657,7 +694,53 @@ def install_repositories(self):
                 t.get('changeset_revision', "")
             ) for t in self.errored_repositories])
         )
-        log.info("All repositories have been processed.")
+        log.info("All repositories have been installed.")
+        if self.test:
+            installed_tools = []
```

Comment:
Maybe this should get its own method? `test_repositories"

Reply:
Agreed - I'll try to make this change.